subjects/road_intersection/README.md

```diff
@@ -101,17 +103,20 @@ For this it must be possible to do the following:
 
 - The `Esc` key must finish the simulation.
 
-> Arrow keys must not let the user spam the creation of vehicles, vehicles must be created with a safe distance between them.
+> Arrow keys must not let the user spam the creation of vehicles, they must be created with a safe distance between them.
+
+> A safe distance is any distance that allows the vehicles not to crash into each other.
 
 ### Example
 
-You can see an example [here](https://www.youtube.com/watch?v=6B0-ZBET6mo).
+You can see an example for the road_intersection [here](https://www.youtube.com/watch?v=6B0-ZBET6mo).
```

Comment:
This shows an example which does not feature a four  lane intersection.
Should we clarify that this only shows traffic lights & collision avoidance?

Reply:
it has for lanes, the example shows 2 roads and four lanes

Reply:
My comment is irrelevant. We're using single carriageways.